website/content/en/preview/upgrading/upgrade-guide.md

```diff
@@ -37,6 +37,144 @@ kubectl apply -f https://raw.githubusercontent.com/aws/karpenter{{< githubRelRef
 WHEN CREATING A NEW SECTION OF THE UPGRADE GUIDANCE FOR NEWER VERSIONS, ENSURE THAT YOU COPY THE BETA API ALERT SECTION FROM THE LAST RELEASE TO PROPERLY WARN USERS OF THE RISK OF UPGRADING WITHOUT GOING TO 0.32.x FIRST
 -->
 
+### Upgrading to `1.0.0`+
+
+{{% alert title="Warning" color="warning" %}}
+`1.0.0`+ supports Karpenter v1 and v1beta1 APIs and will not work with earlier Provisioner, AWSNodeTemplate or Machine alpha APIs. Do not upgrade to `1.0.0`+ without first [upgrading to `0.32.x`]({{<ref "#upgrading-to-0320" >}}) or later. This version supports both the v1beta1 and v1 APIs, allowing you to migrate all of your existing APIs to v1beta1 APIs without experiencing downtime.
+{{% /alert %}}
+
+Before you begin upgrading to `1.0.0`, you should know that:
+
+* Every Karpenter upgrade from pre-v1.0.0 versions must go through an upgrade to minor version `v1.0.0`.
+* You must be on a version of Karpenter that supports NodePools, NodeClaims, and NodeClasses (`0.32.0`+ Karpenter versions support v1beta1 APIs).
+* You must complete all `Before upgrading to 1.0.0` Prerequisites.
+* You must complete all `Before upgrading to `1.1.0` Prerequisites before upgrading to `1.1.0` later. Although v1beta1 yaml files should work for `v1.0.0`, the will fail with the release of `v1.1.0`.
+
+
+#### Prerequisites
```

Comment:
We need to call out that customer will need to update their IAM permissions when upgrading to 1.0